custom_components/fellow/coordinator.py

```
@@ -81,24 +84,38 @@ def __init__(
-        """Use slower safety polling while cloud invalidations are connected."""
-        interval = (
-            PUSH_CONNECTED_POLL_INTERVAL_SECONDS
-            if connected
-            else self._configured_update_interval_seconds
-        )
-        update_interval = timedelta(seconds=interval)
-        if self.update_interval != update_interval:
-            self.update_interval = update_interval
-            if self._listeners:
-                self._schedule_refresh()
+    def set_push_connected(self, _connected: bool) -> None:
```

Ruff (0.16.3)

[warning] Boolean-typed positional argument in function definition

(FBT001)